core/src/yamlscript/runtime.clj

```diff
@@ -54,6 +54,7 @@
 
               ;; clojure.core functions overridden by YS
               'load (sci/copy-var ys.ys/load-file nil)
+              'cpp-compile-file (sci/copy-var ys.ys/cpp-compile-file nil)
```

Comment:
Not sure why this is here.
These are common Clojure functions that I wanted to use their names for specific YS functions.